packages/create-hintrc/package.json

```diff
@@ -10,8 +10,10 @@
     "create-hintrc": "./dist/src/create-hintrc.js"
   },
   "dependencies": {
+    "@hint/utils": "^1.0.0",
     "browserslist": "^4.5.4",
```

Comment:
So this was a missing dependency? I wonder why `lint:dependencies` didn't catch it up 🤔

Reply:
`@hint/utils` is new.